### ISSUE TITLE
Moving a floating window is not possible within a for_window rule

### ISSUE THREAD

**Issue by m1cr0man** [swaywm/sway#1523]
## Issue
Attempting to move a new floating window around using `move <left|right|up|down> <px>` does nothing when applying it from a for_window rule.

Sway version: 0.15.0
Gist (config + log): https://gist.github.com/m1cr0man/21266b66429c7663a2c14dbc5141cdce#file-config-L143

## Steps to reproduce
1. Create a for_window rule similar to this:
```bash
for_window [title="^Skype$"] border pixel 0; floating enable; move down 600; move right 600
```
2. Influence your floating window to spawn. It will not spawn where you expect it to.

## Workaround
Set the size of the window before trying to move it. Example modification to above rule:
```bash
for_window [title="^Skype$"] border pixel 0; floating enable; resize set 288 180; move down 600; move right 600
```

**Comment by tardypad** [comment]
I confirm that problem too
Still happening in version 0.15.2

**Comment by SirCmpwn** [comment]
Can you use the move command on a floating window outside of a rule?

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unrelated sway 1.0.
